src/components/site/tableOfContents/index.jsx

```diff
@@ -40,11 +46,23 @@ class TableOfContents extends React.PureComponent {
         label: link.title,
         to: `${this.TO_PREFIX}${link.slug}`,
         primaryText: link.title,
+        leftIcon: <Check />,
         active: this.setActivePage(link.id),
+        className: classnames(
+          'toc-link',
+          `link--page-id--${link.id}`,
+          this.setActivePage(link.id) ? 'link-active' : '',
```

Comment:
no need to evaluate `setActivePage` twice.  Or rather, if you need a value twice, I'm of the opinion that it's worth setting it to a variable and then just using that variable twice